README.md

```diff
@@ -54,6 +54,8 @@ Go to Stash general settings screen on SonarQube server to fill:
 
 **Stash issue threshold** (sonar.stash.issue.threshold): To limit the number of issue pushed to Stash.
 
+**Stash issue severity threshold** (sonar.stash.issue.severity.threshold): To push issues of specified or higher severity to Stash. Affects only diff comments, overview will still contain all severities. By default, all diff issues are pushed (threshold: INFO).
```

Comment:
readme updated to indicate that this settings will affect only the diff comments, not the summary one.

Reply:
I think we can remove the `(threshold: INFO)`, as it is implicit.

Reply:
I would also prefer the wording:
`Minimum issue severity to create diff-view comments for.` (or something like this)